hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/distributions/smtable/SwModuleTable.java

```diff
@@ -369,30 +323,23 @@ private String getNameAndVerion(final Object itemId) {
         return name + "." + version;
```

Comment:
Only in loops the compiler can optimize string concatenation.
In this case it is not necessary to use a StringBuilder:
1. It is more readable
2. The compiler will do this

"Some people will tell you that String concatenation with + is a performance killer on Java. They’ll tell you to use StringBuilder instead. But javac is actually pretty smart about converting + into StringBuilder app"
https://briangordon.github.io/2014/01/javac-optimizations.html